ranchhand.tf

```diff
@@ -9,7 +9,7 @@ locals {
 }
 
 module "ranchhand" {
-  source = "github.com/dominodatalab/ranchhand.git//terraform?ref=v0.1.2-rc1"
+  source = "github.com/dominodatalab/ranchhand.git//terraform?ref=v0.1.4-rc1"
```

Comment:
@steved The release is yet to be created. Waiting for dominodatalab/ranchhand#32 to be merged.